pkg/addons/data/Service/openshift-branding/branding.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: branding-serving-cert
```

Comment:
drop this - at the point where we use https, we'll use the default signed router wildcard certificate and won't need this